java/code/src/com/redhat/rhn/common/conf/ConfigDefaults.java

```diff
@@ -329,6 +330,11 @@
      */
     public static final String ALLOW_ADDING_PATCHES_VIA_API = "java.allow_adding_patches_via_api";
 
+    /**
+     * Specify the default language to use if user preferences are not available such as on the login page
+     */
+    public static final String PREFERRED_LOCALE = "web.locale";
```

Comment:
why not `DEFAULT_LOCALE`? btw, is this going to affect the language or also all the localization like date and time?

Reply:
I guess you are right `DEFAULT_LOCALE` is better in this context.
It is just the language.